deep_qa/training/multi_gpu.py

```diff
@@ -0,0 +1,89 @@
+from keras.layers import merge
+from keras.layers.core import Lambda
+import keras.backend as K
+import tensorflow
+from .models import DeepQaModel
+
+
+def pin_variable_device_scope(device, parameter_device="/cpu:0"):
```

Comment:
You use `variable` in the method name, but `parameter` in the argument name.  These should be consistent.